udata/search/fields.py

```diff
@@ -56,7 +58,20 @@ def get_value_filter(self, value):
 
 
 class TermsFacet(Facet, DSLTermsFacet):
-    pass
+
+    def add_filter(self, filter_values):
+        if not filter_values:
```

Comment:
After a few attempts, it doesn't work because you can't return lists/iterators here.

Let's do that the other way:

1. which cases should we cover? addressed to whom?
2. add tests related to these cases
3. implement the solution 😄